### ISSUE TITLE
Class '_Union' has no '__args__' member (no-member)

### ISSUE THREAD

**Issue by SamyCookie** [PyCQA/pylint#2419]
Hi,
From this code :
```
from typing import Union
NumericTypes = Union[int, float, Decimal]
i = 42
print( isinstance(i, NumericTypes.__args__) )
```

I got a false positive:
```
E1101: Class '_Union' has no '__args__' member (no-member)
```

```
$ ~/.local/bin/pylint --version
pylint 2.1.1
astroid 2.0.4
Python 3.5.3 (default, Jan 19 2017, 14:11:04) 
[GCC 6.3.0 20170118]
```

Have a good day !

**Comment by PCManticore** [comment]
Thanks for reporting an issue, I can also reproduce it.

**Comment by SamyCookie** [comment]
awesome ! thanks to fix that bug !
